source/jormungandr/jormungandr/parking_space_availability/bss/tests/cykleo_test.py

```diff
@@ -61,6 +72,16 @@ def parking_space_availability_cykleo_support_poi_test():
     poi['properties']['operator'] = 'Bad_operator'
     assert not provider.support_poi(poi)
 
+    # same test with "02" in id, not "2"
+    assert provider.support_poi(poi_with_0)
+    poi_with_0['properties']['operator'] = 'Bad_operator'
```

Comment:
you are modifying a global var which is shared in other tests

prefer deep-copying before you modify it

Reply:
Yep, just managed it the same way it was done before... I just changed that.